code/game/objects/items/devices/radio/radio.dm

```diff
@@ -39,6 +39,8 @@
 	var/syndie = FALSE  // If true, hears all well-known channels automatically, and can say/hear on the Syndicate channel.
 	var/list/channels = list()  // Map from name (see communications.dm) to on/off. First entry is current department (:h).
 	var/list/secure_radio_connections
+	var/radio_silent = FALSE // If true, radio doesn't make sound effects (ie for Syndicate internal radio implants)
+	var/list/radio_sounds = list('sound/effects/radio1.ogg','sound/effects/radio2.ogg') //Sephora - Radios make small static noises now
```

Comment:
Remove the list and just `pick()` these two where they're used.

Readability < memory usage. Plus you shouldn't init a list at the type declaration level anyways.

Reply:
Fixed